src/k8s/pkg/k8s/client/cluster.go

```diff
@@ -23,10 +24,15 @@ func (c *k8sdClient) IsBootstrapped(ctx context.Context) bool {
 // Bootstrap bootstraps the k8s cluster
 func (c *k8sdClient) Bootstrap(ctx context.Context, bootstrapConfig apiv1.BootstrapConfig) (apiv1.NodeStatus, error) {
 	// Get system hostname.
-	hostname, err := os.Hostname()
+	rawHostname, err := os.Hostname()
 	if err != nil {
 		return apiv1.NodeStatus{}, fmt.Errorf("failed to retrieve system hostname: %w", err)
 	}
+	// TODO: this should be done on the server side, but we cannot currently hijack the microcluster bootstrap endpoint.
```

Comment:
could you add a note to https://warthogs.atlassian.net/browse/KU-455 to remember this? Thanks